backend/src/models/user.js

```diff
@@ -17,7 +18,25 @@ const UserSchema  = new mongoose.Schema({
         type: String,
         required: true,
         unique: true
-    }
+    },
+    firstName: {
+        type: String,
+        required: true
+    },
+    surname: {
+        type: String,
+        required: true
+    },
+    phone: String,
+    picture: {
+        large: {data: Buffer, imageType: String},
+        thumbnail: {data: Buffer, imageType: String}
+    },
+    certificate: {
+        data: Buffer, imageType: String
+    },
+    isInstructor: Boolean,
```

Comment:
Please pull from Martins branch before, we already add this and it has different name.